src/pages/auth/Signup/signup.tsx

```diff
@@ -124,6 +128,25 @@ function SignUp() {
                 </p>
               )}
             </div>
+            <div className="mb-5 grid">
+              <input
+                {...register("phone_number", {
+                  required: true,
+                  min: 2,
+                  max: 25,
+                  pattern: /^\S+@\S+$/i,
+                })}
+                style={{ borderColor: errors.phone_number ? "#b92828" : "" }}
+                className="bg-[#F1F3F7] dark:bg-[#1E1E1E] border-[#DCDDE1] dark:border-[#353535] rounded-sm border-[1.8px] h-[40px] w-[20rem] placeholder:text-[14px] dark:placeholder:text-[#353535] placeholder:text-[#5D6675] pl-4 focus:outline-none dark:focus:border-white focus:border-[#3D4450]"
+                type="tel"
+                placeholder="Phone number"
+              />
+              {errors.phone_number && (
+                <p className="text-[#b92828] text-[12px]">
+                  Phone must be valid {errors.phone_number?.message}
```

Comment:
What do you think about making this "Phone number must be valid"?

Reply:
phone number is more descriptive, "phone must be valid" makes it sound like the person might be using a phone with is not valid🤣